code/model/SubgoalPolicy.cpp

```diff
@@ -1099,8 +1107,13 @@ bool SubgoalPolicy::LoadSimpleConnectionFile (String filepath)
 
 
 //													
-bool SubgoalPolicy::LoadFeatureConnectionFile (String filepath)
+bool SubgoalPolicy::LoadFeatureConnectionFile (String filepath, bool update)
```

Comment:
if update is true, I am updating, otherwise it means it is the first time I am running this, so I am starting setting up